index.ts

```diff
@@ -96,6 +109,7 @@ if (syncRepoLabels) {
 
 const payload = require(process.env.GITHUB_EVENT_PATH as string)
 const number = (payload.issue || payload.pull_request || payload).number
+const action: string = payload.action
```

Comment:
maybe you could refactor the remaining `payload.action` mentionings in line 237, 213, 169

Reply:
Nice catch, refactored.